New Compilation warnings from output layout

We had no prior warnings. Can we solve this?

```
[199/296] Compiling C++ object src/liblibwayfire.a.p/core_output-layout.cpp.o
../src/core/output-layout.cpp: In member function ‘wf::output_state_t wf::output_layout_output_t::load_configured_state()’:
../src/core/output-layout.cpp:542:26: warning: ‘tmp.wlr_output_mode::picture_aspect_ratio’ may be used uninitialized [-Wmaybe-uninitialized]
  542 |             state.mode   = (is_mode_supported(tmp) ? tmp : select_default_mode());
      |             ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/core/output-layout.cpp:528:25: note: ‘tmp.wlr_output_mode::picture_aspect_ratio’ was declared here
  528 |         wlr_output_mode tmp;
      |                         ^~~
../src/core/output-layout.cpp:542:26: warning: ‘tmp.wlr_output_mode::link.wl_list::prev’ may be used uninitialized [-Wmaybe-uninitialized]
  542 |             state.mode   = (is_mode_supported(tmp) ? tmp : select_default_mode());
      |             ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/core/output-layout.cpp:528:25: note: ‘tmp.wlr_output_mode::link.wl_list::prev’ was declared here
  528 |         wlr_output_mode tmp;
      |                         ^~~
../src/core/output-layout.cpp:542:26: warning: ‘tmp.wlr_output_mode::link.wl_list::next’ may be used uninitialized [-Wmaybe-uninitialized]
  542 |             state.mode   = (is_mode_supported(tmp) ? tmp : select_default_mode());
      |             ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/core/output-layout.cpp:528:25: note: ‘tmp.wlr_output_mode::link.wl_list::next’ was declared here
  528 |         wlr_output_mode tmp;
      |                         ^~~
```

--- a/src/core/output-layout.cpp
+++ b/src/core/output-layout.cpp
@@ -525,7 +525,7 @@ struct output_layout_output_t
         state.position = position_opt;
 
         wf::output_config::mode_t mode = mode_opt;
-        wlr_output_mode tmp;
+        wlr_output_mode tmp{};
 
         switch (mode.get_type())
         {

